src/Trinity.TSL/Trinity.TSL.Parser/tb_semantc_check.cpp

```diff
@@ -42,6 +42,8 @@ inline string get_key(NProtocolProperty* target)
     case PT_STRUCT_RESPONSE:
         return "NProtocolProperty::ResponseType";
         break;
+      case PT_UNDEFINED: break;
```

Comment:
these cases are covered in the throw.
please revert.

Reply:
statement reverted